modules/core/04-channel/keeper/keeper.go

```diff
@@ -384,6 +384,24 @@ func (k Keeper) IterateChannels(ctx sdk.Context, cb func(types.IdentifiedChannel
 	}
 }
 
+// GetChannelsWithPortPrefix returns all channels with the specified port prefix.
+func (k Keeper) GetChannelsWithPortPrefix(ctx sdk.Context, portPrefix string) []types.IdentifiedChannel {
```

Comment:
Should we consider naming this function to align with `GetAllChannels` below which returns `[]types.IdentifiedChannel` also?

Maybe `GetAllChannelsWithPortPrefix` or `GetAllPortPrefixedChannels`? 

I don't feel particularly strongly but just a thought. cc. @crodriguezvega @colin-axner

Reply:
I like `GetAllChannelsWithPortPrefix` to stay consistent

Reply:
Sounds good to me. I have no preference